pkg/manager/manager.go

```diff
@@ -5,13 +5,19 @@ import (
 	"net"
 
 	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/gofrs/flock"
 	"github.com/rs/zerolog/log"
 	"github.com/vishvananda/netlink"
+	nl "github.com/vishvananda/netlink/nl"
 
 	"github.com/k8snetworkplumbingwg/accelerated-bridge-cni/pkg/types"
 	"github.com/k8snetworkplumbingwg/accelerated-bridge-cni/pkg/utils"
 )
 
+const (
+	vlanUplinkLockFile = "/tmp/accelerated-bridge-cni.lock"
```

Comment:
we already use `/var/lib/cni/accelerated-bridge/` to store cni state for pods.
IMO we should use the same path for lock file as well.
(will require to ensure the path exists before attempting to lock)

Reply:
Agree.  Ill create a wrapper around flock.Flock to take care of setting the path up.